src/app/core/components/header/header.component.ts

```diff
@@ -1,10 +1,38 @@
-import { Component, OnInit, Input } from '@angular/core';
+import { Component, OnInit, Input } from "@angular/core";
+import { TimeTableService } from "../../../timetable/services/time-table.service";
 
 @Component({
-  selector: 'app-header',
-  templateUrl: './header.component.html',
-  styleUrls: ['./header.component.scss']
+  selector: "app-header",
+  templateUrl: "./header.component.html",
+  styleUrls: ["./header.component.scss"]
 })
 export class HeaderComponent {
-  @Input() weekDay: string;
+  weekDay: string = "Monday";
+  constructor(private _timetableService$: TimeTableService) {}
```

Comment:
1. It's better to write `timetableService` instead of `_timetableService$`. In angular is deprecated the use of `_` and `$` is put only for observables
2. put a new line after and before constructor